storage/postgres/abstract.go

```diff
@@ -104,7 +104,7 @@ func columnsByTags(tags []tagType) map[string]bool {
 
 func validateFieldQueryParams(columns map[string]bool, criteria []query.Criterion) error {
 	for _, criterion := range criteria {
-		if criterion.Type == query.FieldQuery && !columns[criterion.LeftOp] {
+		if !columns[criterion.LeftOp] {
```

Comment:
method still receives Criterion type so it should handle properly cases when a non field query is passed (for example ignore criterias with different type)

Reply:
Maybe we need to return the check, as i don't remember a reason for removing it.